### ISSUE TITLE
less 661 --with-secure: ld: symbol(s) not found for architecture arm64

### ISSUE THREAD

**Issue by mytec33** [gwsw/less#544]
Hi,

When attempting to build less from a tarball release with --with-secure configured, the build process fails on macOS.

```
...
gcc -I. -c -DBINDIR=\"/opt/billbrew/bin\" -DSYSDIR=\"/opt/billbrew/etc\"  -g -O2 version.c
gcc -I. -c -DBINDIR=\"/opt/billbrew/bin\" -DSYSDIR=\"/opt/billbrew/etc\"  -g -O2 xbuf.c
gcc  -o less main.o screen.o brac.o ch.o charset.o cmdbuf.o command.o cvt.o decode.o edit.o evar.o filename.o forwback.o help.o ifile.o input.o jump.o line.o linenum.o lsystem.o mark.o optfunc.o option.o opttbl.o os.o output.o pattern.o position.o prompt.o search.o signal.o tags.o ttyin.o version.o xbuf.o   -lncurses
Undefined symbols for architecture arm64:
  "_add_content_table", referenced from:
      _init_cmds in decode.o
      _init_cmds in decode.o
  "_lesskey_content", referenced from:
      _init_cmds in decode.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [less] Error 1
```

Apple clang version 15.0.0 (clang-1500.3.9.4)
Target: arm64-apple-darwin23.5.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

ProductName:		macOS
ProductVersion:		14.5
BuildVersion:		23F79

**Comment by gwsw** [comment]
Thanks. Fixed in 26c7a12154ae4f84e60840129684831b27701b96.

**Comment by mytec33** [comment]
Thank you, less now builds without error.
